beets/ui/commands.py

```diff
@@ -1324,8 +1324,21 @@ def stats_func(lib, opts, args):
 # version: Show current beets version.
 
 def show_version(lib, opts, args):
+    # Use the improved 'distro' package for platform info, if available
+    # (the stdlib platform module has simplistic logic, and 3.7 will drop it)
+    from platform import platform
+    os_id = platform()
+    if os.name == 'posix':
+        try:
+            import distro
```

Comment:
Do you have a sense of how likely it is for `distro` to be installed? Is there some fallback way we can get some evidence about what the platform is, even if it’s not as nice as this module’s output?